### PR TITLE
Write/ability to delete saved covers

### PR DESCRIPTION
**Pull request by angelewilliams**
Hey @jordan-sullivan! 
So excited that we go this working. 

- Summary: We wrote the deleteSavedCover function, and connected the ondblclick event into the html so that a user can delete a saved cover upon double click. 
- No major issues that were caught, maybe test switching between pages a few times. 
- Please look at main.js  lines 34-41 and 47-8 for the code that executes our functionality goal.

Thanks!
Angele

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -18,16 +17,9 @@ var saveInputTitle = document.querySelector(".user-title");
...
 
-var savedCovers = [
-  new Cover(
-    "http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg",
-    "Sunsets and Sorrows",
-    "sunsets",
-    "sorrows"
-  ),
-];
+var currentCover = new Cover();
```
Comment (jordan-sullivan) [inline src/main.js]: Please add lines 23 and 24 to top of page as they are global variables.

**Review by jordan-sullivan — CHANGES_REQUESTED** [review]
Hi @angelewilliams , project is looking great so far. Before merging, please complete the following:
- move the 2 global variables to top of page. 
- delete all comments from main.js file.

When refactoring in next section of project with your partner, keep in mind:
- ensuring all punctuation is consistent
- refactoring hide and view Elements as functions
- adding in your README file

File: src/main.js (hunk 1 above)
Comment (angelewilliams) [reply src/main.js]: Done!

**Comment by angelewilliams** [comment]
Hi @jordan-sullivan,
 I just made the changes you requested and pushed back to this branch. Please review and merge if everything looks okay!

**Pull request merged**